Journal check: handle cases when there are no entries

jlausuch opened the pull request:
If there are no error messages in the journal, it will print
"-- No entries --" message, which is considered by the test module
as a failure. We should filter those cases.

- Related ticket: https://progress.opensuse.org/issues/101984

DimStar77 commented:
Maybe filtering as part of an ignore in bug_refs.json could be considered a bit cleaner than an extra grep call

Vogtinator commented:
Yes, or passing the `-q` option to hide all the non-log messages. That probably drops the boot markers though.

jlausuch commented:
> Maybe filtering as part of an ignore in bug_refs.json could be considered a bit cleaner than an extra grep call

I can do that, but keep in mind that we will need to update the bugref for new version and will miss the first time it occurs.. If this is a generic "false negative" case, it would be better to handle these kind of situations in the test code.

DimStar77 commented:
> Yes, or passing the `-q` option to hide all the non-log messages. That probably drops the boot markers though.

indeed, the bookmakers vanish with -q as well (tested that before too)

jlausuch commented:
> Yes, or passing the `-q` option to hide all the non-log messages. That probably drops the boot markers though.

Updated with this suggestion. I have also removed the `tail` as I guess it was just to filter out the same type of output.
